### DISCUSSION TITLE
Server could not start by unsupported Nod.js

### DISCUSSION THREAD

**Discussion by ErythroCraft** [requarks/wiki#6623] [Error / Bug Report]
Hello

I installed the wiki server and would like start it by using the [ node server ] cmd.

But PowerShell generates this Error:

    PS C:\xampp\htdocs\minecraft\wiki> node server
    You're using an unsupported Node.js version. Please read the requirements.
    PS C:\xampp\htdocs\minecraft\wiki>

My system structure is using an self hosted xammp/apache website/server with MySQL at Win11 ipsec server

Node version: 20.5.0-x64
Wiki.js version: 2.5.299

OS: Win 11

**Comment by P4mes4n** [comment]
https://docs.requarks.io/install/requirements#nodejs

**Comment by ErythroCraft** [comment]
Thanks for the info.
